flows/atd-finance-data/finance_data_to_s3.py

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env python
+
+"""
+Name: ATD Finance Data Flow
+Description: Gets Finance data from a database, places it in an S3 bucket, 
+             then moves it along to Knack and socrata.
+
+Build Deployment yaml file:
+$ prefect deployment build flows/atd-finance-data/finance_data_to_s3.py:main \
+--name "Finance Data Publishing" \
+--cron "13 7 * * *" \
+--pool atd-data-03 -q default \
+-sb github/finance-data-wip \
+-o "deployments/finance_data_to_s3.yaml" \
+--description "Repo: https://github.com/cityofaustin/atd-finance-data, Gets Finance data from a database, places it in an S3 bucket, then moves it along to Knack and socrata."
```

Comment:
Pointing out in the latest Prefect release they added a `--description` CLI argument!